grafana/meta/collectd.yml

```diff
@@ -0,0 +1,10 @@
+{%- if pillar.grafana.server is defined %}
+{%- from "grafana/map.jinja" import server with context %}
+
+{%- if server.enabled %}
```

Comment:
more robust and coherent w/ other states

{%- if server.get('enabled', False) %}